ScientistProfile.create(:id => 17, :department => "Physical Biosciences", :department_url => "http://pbd.lbl.gov/index.php", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Evans Hall", :city => "Berkeley", :state => "CO", :zip_code => 94720, :phone1 => "510-643-5624", :phone2_type => "Fax", :phone2 => "510-664-4115", :email => "fletch@berkeley.edu", :emphasis => "<p>The Fletcher Lab is developing a toolbox of new instruments and techniques &ndash; optical, mechanical, fluidic &ndash; that let us explore the &ldquo;design rules&rdquo; of cells and their molecular components. By learning how cells are built to physically interact with their environment, we aim to understand the operation of molecular machines that drive cell movements, identify physical mechanisms of disease, and develop medical devices for diagnosis and intervention of current health problems. In the future, the tools that help us understand how cells are engineered will also help us re-engineer cell function to address modern technology needs.</p>\n<p><strong>Optical &amp; force microscopy:</strong> Optical microscopy has been and remains the primary method for studying the molecular organization and dynamics of living cells. However, pictures must be coupled with biophysical measurements to fully explain processes such as cell motility, cytokinesis, and phagocytosis. Analysis of cell and molecular mechanics requires spatial resolution on the scale of nanometers, force resolution below a nanonewton, and temporal resolution and range from milliseconds to hours. We are developing instruments based on the atomic force microscope (AFM) that combine the capabilities of force microscopy with advantages of optical imaging.</p>\n<p><strong>Molecular mechanics:</strong> Cells have been described as a collection of protein machines. Highly regulated conformational changes in proteins driven by nucleoside triphosphate hydrolysis, ion gradients, or other energy sources control the complex series of reactions that enable all cell functions. The cytoskeletal protein actin exerts forces necessary for a wide range of cell movements by assembling non-covalent polymers in the form of single filaments, stabilized bundles, and cross-linked meshes. We are studying the mechanical properties of actin and force generation by actin polymerization to understand the operation of this ubiquitous actuator.</p>\n<p><strong>Cell mechanics &amp; disease:</strong> Mechanical properties of cells arise from the dynamic behavior of the cytoskeleton and physical interactions with the environment. Properties such as cell stiffness and adhesion play a vital role in normal physiological function. These and other properties can be disrupted by the onset of disease and may even contribute to the progression of disease. We are developing measurement techniques to quantify the role of mechanical properties in complications associated with cancer and pathogen infection.</p>\n<p><strong>Medical devices:</strong> Medical devices utilize innovations in technology and advancements in biology to improve diagnosis and treatment of disease. We are developing devices to solve medical problems related to cell and tissue mechanics. Effective delivery of therapeutic drugs into the bloodstream is essential for the treatment of many medical problems. As pharmaceutical companies develop more complex macromolecules for gene therapy, enzyme replacement, and disease immunization, conventional drug delivery techniques will no longer be sufficient. We are developing a new tool for controlled delivery of drugs in the body based on a high-speed liquid microjet.</p>", :scientist_id => 23, :publications_url => "http://fletchlab.berkeley.edu/node/1")
ScientistProfile.create(:id => 22, :department => "Physical Biosciences Division", :department_url => "http://pbd.lbl.gov/", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "461A Koshland Hall", :city => "Berkeley ", :state => "CA", :zip_code => 94720, :phone1 => "510-643-9131", :phone2_type => "Fax", :phone2 => "208-279-8978", :email => "SEBrenner@lbl.gov", :emphasis => "<p>We develop computational methods for the analysis and integration of molecular sequence, structure, and expression data. Our aim is to understand organismal biology by interpreting the information encoded in complete genomes. This work is presently focused on the areas of structural and functional genomics. Structural genomics projects attempt to provide an experimental structure or a good theoretical model for every tractable protein in all completed genomes. Our work involves organizing proteins into families according to homology; classifying proteins and RNA according to structure; predicting structure from homology and constructing atomic coordinate models; providing information resources for structural genomics; developing methods for selection of proteins for experimental characterization; and analyzing solved structures to detect homology and functional information.</p>\n<p>We study computational functional genomics by creating algorithms using molecular sequence, structure, phylogeny, and expression information to infer the functions of genes. This work includes the use of gene genealogies to trace gene histories and functional divergences; reverse-genomics comparison of multiple complete genomes to locate genes associated with characterized cellular or biochemical functions; creation of databases of genomic information; and continued refinement of sequence comparison methods. We also combine sequence comparison with expression and other experimental data to improve molecular and cellular functional characterization.</p>", :scientist_id => 7, :publications_url => "http://compbio.berkeley.edu/people/brenner/pubs/")
ScientistProfile.create(:id => 21, :department => "Biochemical Engineering", :department_url => "http://cheme.berkeley.edu/", :company => "University of California, Berkeley", :address1 => "Department of Biochemical Engineering", :address2 => "491 Tan Hall MC 1462", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-642-1387", :phone2_type => "Secondary Lab Phone", :phone2 => "510-486-7069 ", :email => "hwblanch@lbl.gov", :emphasis => "<p>Current research in the Blanch lab is focused on three areas:</p>\n<p>1. Protein interactions: The broad objectives of this research are to develop molecular-thermodynamic descriptions of the behavior of proteins in electrolyte solutions, to provide a framework for the design and optimization of protein separation systems, in particular protein separation by precipitation and protein crystallization.</p>\n<p>2. DNA Electrophoresis: By observing single-molecule DNA-polymer entanglements directly as DNA electrophoreses through a capillary, we see that DNA/polymer collisions depend on the size and concentration of each species, and that multiple entanglements are possible. We have obtained data on the entanglement time as a function of the Porod-Kratky persistence length of the polymer (a measure of its stiffness), the collision frequency, and probability distribution of multiple entanglements. Using these data, a mechanistic model for DNA separations has been developed. It is able to predict DNA separations as a function of DNA size and polymer properties.</p>\n<p>3. Mammalian Cell Metabolism: While conditions for growth of bacterial, yeast and fungal systems are well established, much less is known about the growth, physiology and regulation of mammalian cells. In earlier work, we studied the environmental factors important in hybridoma growth, and developed an NMR-based method to follow intracellular metabolites in a hollow fiber bioreactor, placed in the bore of the NMR magnet. This has been continued to study glutamine metabolism in a murine hybridoma in detail. The understanding so obtained has been valuable in guiding industrial bioreactor operation for production of monoclonal antibodies and other products. The data and methodology developed now permits the rational design of balanced media, feed strategies and reactor operation to optimize antibody yields, and the approach is now common to most companies producing products from hybridoma, CHO or BHK cell lines.</p>", :scientist_id => 6, :publications_url => "http://cheme.berkeley.edu/faculty/blanch/")
ScientistProfile.create(:id => 18, :department => "Natural & Artificial Photosynthesis", :department_url => "http://pbd.lbl.gov/index.php", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Building 66", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-486-4325", :email => "HMFrei@lbl.gov", :summary => "<p>The goal of our research is to develop robust artificial photosynthetic systems for the synthesis of fuels and chemicals from carbon dioxide and water using sunlight as energy source. To take advantage of the flexibility and precision by which light absorption, charge transport and catalytic properties can be controlled by discrete molecular structures, we are assembling well-defined inorganic polynuclear units in nanoporous silica scaffolds that function as visible light photocatalysts for carbon dioxide reduction or water oxidation. Photocatalytic units anchored on silica nanopore surfaces consist of an oxo-bridged binuclear metal-to-metal charge-transfer chromophore that acts as a visible light electron pump coupled to a multi-electron transfer catalyst (Figure). Selective synthetic methods are being developed that afford all-inorganic units with donor and acceptor metal centers with selectable redox potential, thus offering a means to achieve desirable light absorption properties and thermodynamic efficiency. The large surface area of the nanoporous silica supports affords the high density of photocatalytic sites needed for the chemistry to keep up with the solar flux, while the nanostructured features offer opportunities for separating the reduction from the oxidation half reaction in an integrated photosynthetic system. Photocatalysts for the efficient oxidation of water to O2 by visible light and for carbon dioxide splitting to CO have been demonstrated.</p>\n<p>Emphasis in our work is on the structural elucidation of the photocatalytic units by EXAFS, XANES, FT-Raman, FT-IR, EPR and optical spectroscopy. High resolution transmission electron microscopy and diffraction measurements reveal the structure of metal oxide nanoclusters that serve as multi-electron catalyst inside the silica mesopores. Electron transfer processes of the photocatalytic units are studied by transient optical absorption spectroscopy,while the mechanisms of the water and carbon dioxide activation steps are explored by time-resolved FT-IR spectroscopy on the nanosecond to millisecond time scale. We have been able to detect transient surface intermediate at the gas-nanopore and liquid-nanopore interface using transmission or attenuated total reflection methods. The mechanistic understanding, combined with knowledge of electronic and structural details of the polynuclear photocatalysts are a key ingredient for improving the design of the systems.</p>", :emphasis => "<p>Recent Publications:</p>\n<p>N. Sivasankar, W.W. Weare, and H. Frei.&nbsp; Direct Observation of a Hydroperoxide Surface Intermediate upon Visible Light Sensitized Water Oxidation at Ir Oxide Nanocluster Catalyst by Rapid-Scan FT-IR Spectroscopy. <em>J. Am. Chem. Soc.</em> <strong>133</strong>, 12976-12979(2011).</p>\n<p>H.S. Soo, M.L. Macnaughtan, W.W. Weare, J. Yano, and H. Frei.&nbsp; EXAFS Spectroscopic Analysis of Heterobinuclear TiOMn Charge-Transfer Chromophore in Mesoporous Silica.&nbsp; <em>J. Phys. Chem. C</em>, submitted.</p>\n<p>N. Sivasankar and H. Frei.&nbsp; Direct Observation of Kinetically Competent Surface Intermediates upon Ethylene Hydroformylation over Rh/Al2O3 under Reaction Conditions by Time-Resolved FT-IR Spectroscopy. <em>J. Phys. Chem. C</em> <strong>115</strong>, 7545-7553 (2011).</p>\n<p>T. Cuk, W.W. Weare, and H. Frei.&nbsp; Unusually Long Lifetime of Excited Charge-Transfer State of Binuclear TiOMnII Unit Anchored on Silica Nanopore Surface. <em>J. Phys. Chem. C</em> <strong>114</strong>, 9167-9172 (2010).</p>\n<p>F. Jiao and H. Frei.&nbsp; Nanostructured Manganese Oxide Clusters Supported on Mesoporous Silica as Efficient Oxygen-Evolving Catalysts. <em>Chem. Commun. </em><strong>46</strong>, 2920-2922 (2010).</p>\n<p>F. Jiao and H. Frei. Nanostructured Cobalt and Manganese Oxide Clusters as Efficient Water Oxidation Catalysts. <em>Energy Environ. Sci. </em><strong>3</strong>, 1018-1027 (2010).</p>\n<p>H. Frei.&nbsp; Polynuclear Photocatalysts in Nanoporous Silica for Artificial Photosynthesis. <em>Chimia</em> <strong>63</strong>, 721-730 (2009).</p>\n<p>F. Jiao and H. Frei.&nbsp; Nanostructured Cobalt Oxide Clusters in Mesoporous Silica as Efficient Oxygen-Evolving Catalysts. <em>Angew. Chem. Int. Ed.</em><strong>48</strong>, 1841-1844 (2009).</p>\n<p>X. Wu, W.W. Weare, and H. Frei.&nbsp; Binuclear TiOMn Charge-Transfer Chromophore in Mesoporous Silica. <em>Dalton</em><em> Trans.</em>, 10114-10121 (2009) (Hot Article).</p>\n<p>W. Weare and H. Frei.&nbsp; Artificial Photosynthesis, In: <em>Yearbook of Science and Technology; </em>Weil, J., Ed.; McGraw Hill: New York (2009); pp.28-31.</p>", :scientist_id => 24)
ScientistProfile.create(:id => 74, :department => "Physical Biosciences", :department_url => "http://pbd.lbl.gov", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Mailstop: 6R2100", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-495-2697", :email => "sclassen@lbl.gov", :emphasis => "<p>In process of writting. Coming soon</p>", :scientist_id => 16)
ScientistProfile.create(:id => 23, :department => "Physical Biosciences Division", :department_url => "http://pbd.lbl.gov/", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "173 Birge Hall", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-643-9706", :phone2_type => "Secondary Lab Phone", :phone2 => "510-642-2163/2-6479", :email => "CJBustamante@lbl.gov", :summary => "<p>Our research is focused on the structural characterization of nucleo-protein assemblies. The structure of chromatin and the global structure of protein-nucleic acid complexes relevant to the molecular mechanisms of control of transcription in prokaryotes are investigated using high resolution scanning force microscopy (SFM). This microscope, also known as Atomic Force Microscope (AFM) works by scanning a tip over the sample to sense the topography of the surface, thus functioning in much the same way than old record players. In addition, we are studying the elastic response of long linear polymers, the forces responsible for maintaining the tertiary structure of proteins, and the mechanical properties of molecular motors, using methods of single molecule manipulation such as laser tweezers and the SFM.</p>", :emphasis => "<p>Our laboratory is involved in the study of the structural basis of protein-DNA interactions and their relevance in the processes of control of gene expression. In prokaryotes, and specially in eukaryotes, replication and transcription regulation involve the interaction of many specialized protein factors at regulator locations on the sequence to insure correct sequence recognition, initiation, processivity, fidelity, and kinetic control. We wish to understand the multiple structural, spatial, and functional relationships among these regulatory factors. We are using the SFM as a high resolution tool to image initiation and elongation transcription complexes of E. coli RNA polymerase to characterize the spatial relationships between the enzyme and the DNA template. We are also beginning to investigate what structural changes are negotiated between RNA polymerase and chromatin during transcription. To this end, we are using the SFM to image complexes of nucleosome-containing DNA fragments carrying a promoter and a terminator upstream and downstream of the nucleosome positioning sequence, respectively. We plan to compare the behavior of various prokaryotic and eukaryotic polymerases as they transcribe through the nucleosome, to investigate whether transcription through a nucleosome is an inherent property of the core particle, or a property of each enzyme itself, and to characterize various intermediates of the translocation process.</p>\n<p>Our laboratory is also working actively in the development of methods of single-molecule manipulation, including the use of SFM cantilevers, optical or laser tweezers, and magnetic beads to investigate the mechanical properties of macromolecules. In one project, we first tether a single protein molecule of T-4 lysozyme between a surface and the end of an SFM cantilever. We can then separate the surfaces in a controlled fashion to induce the mechanical unfolding of the molecule to characterize the nature, range, and strength of the forces that maintain its three-dimensional structure. Our objective is to carry out the unfolding of the molecule at equilibrium so as to obtain the potential energy function of the molecule as a function of the mechanical extension. This function represents the most complete description of the folded state of the protein. We plan to investigate how external conditions in the medium, i.e. temperature, denaturant concentration, etc., or point-directed mutations affect the shape of the potential energy function.</p>\n<p>Finally, our laboratory is also engaged in the study of DNA-binding molecular motors (RNA polymerase, DNA polymerase, etc.) using optical tweezers to investigate the dynamics of these molecules during translocation, as well as the effect of external force load and nucleotide tri-phosphate concentration on their power and force generation. In parallel, we are developing both microscopic (chemical ratchet-type) and phenomenological models of molecular motors which will be tested experimentally.</p>\n<p>We believe that single molecule experiments can provide a unique look into the molecular mechanisms responsible for the mechano-chemical conversion process in these protein machines.</p>", :scientist_id => 8, :publications_url => "http://alice.berkeley.edu/content/publist.php")
ScientistProfile.create(:id => 27, :department => "Physical Biosciences", :department_url => "http://pbd.lbl.gov/", :company => "Lawerence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Hildebrand Hall", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-643-9153", :email => " \tNSGinsberg@LBL.gov", :summary => "LBNL's <em>Today at Berkeley Lab</em> - <a href=\"http://www.lbl.gov/publicinfo/newscenter/tabl/2009/october/10-05-09/\" target=\"_blank\">Meet a Postdoc Spotlight Article</a>\n<p>Postdoctoral scholars are the lifeblood of any research institution. Too often, though, they work in relative anonymity before heading off to their own scientific careers across the country and around the world. The \"Meet a Postdoc\" video series turns some much-needed light on this important segment of Berkeley Lab's population and offers a chance to learn more about them as individuals while they are in our midst. Today&rsquo;s subject is <a href=\"http://www.lbl.gov/publicinfo/newscenter/tabl/2009/october/10-05-09/ginsberg.html\">Naomi Ginsberg</a>, with the Physical Biosciences Division.</p>", :emphasis => "<p><em>Physical and biophysical chemistry; light harvesting, spectroscopy, and imaging</em>.</p>\n<p>Work in the Ginsberg Group is motivated by the need to spatially and temporally resolve the complex dynamics of nanoscale processes, such as photosynthetic light harvesting. We use multiple approaches, separately and in combination, including ultrafast spectroscopy, light microscopy, and cathodoluminescence electron microscopy. &nbsp;A common theme is to deeply understand the nature of the light-matter interactions being used in order to optimally measures these complex dynamics. &nbsp;Current projects aim to (1)&nbsp;map spatio-temporal photoexcitation trajectories onto the architecture of natural and artificial photosynthetic light harvesters by adapting fluorescence microscopy to time-resolve ultrafast energy fl ow, and (2) to develop near-field optical microscopies by leveraging the high spatial resolution of electron optics and the spectral selectivity of cathodoluminescence in order to studying photosynthetic membrane reorganization under physiological conditions.</p>", :scientist_id => 25, :publications_url => "https://sites.google.com/site/nsggrp/pubs")
ScientistProfile.create(:id => 16, :department => "Physical Biosciences", :department_url => "http://pbd.lbl.gov/index.php", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road, MS", :address2 => "201 Hildebrand", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-643-2735", :phone2_type => "Fax", :phone2 => "510-642-6340", :email => "grfleming@lbl.gov", :summary => "<p>Graham Fleming currently serves as UC Berkeley&rsquo;s Vice Chancellor for Research, a position which he assumed in April 2009. Fleming served as the Deputy Director of Lawrence Berkeley National Laboratory from 2005 through 2007. &nbsp;Through joint appointments as Melvin Calvin Distinguished Professor of Chemistry at UC Berkeley, and Founding Director of both the Berkeley Lab's Physical Biosciences Division and UC Berkeley&rsquo;s California Institute for Quantitative Biosciences (QB3), he has re-shaped the intersection of physical and biological sciences, while maintaining his own investigations into ultrafast chemical and biological processes, in particular, the primary steps of photosynthesis. Throughout his administrative career, Fleming has remained a highly active scientific researcher. &nbsp;He has authored or co-authored more than 400 publications, and is widely considered to be one of the world's foremost authorities on ultrafast processes.&nbsp; In addition to his many other activities, Fleming has given numerous talks around the world on the inter-relation and inter-complexity of energy, climate and photosynthesis.&nbsp; In 2007, Fleming led the effort (with co-chair Mark Ratner) to define Grand Challenges in Basic Energy Science for DOE/BES, resulting in &ldquo;Directing Matter and Energy: Five Challenges for Science and the Imagination.&rdquo;</p>", :emphasis => "<p>Ultrafast spectroscopy of chemical and biological systems.<br /> Fleming's work has three general areas: chemical dynamics in liquids and solutions, the dynamics of elementary biological processes, and the development of new types of spectroscopies to reveal molecular dynamics in greater detail. He combined computer simulation and femtosecond fluorescence spectroscopy in a way that led to a deep understanding of the molecular mechanism by which solutes, including dyes and natural pigments such as chlorophylls and carotenoids, interact with their environments (which could be polar liquids such as water and alcohols, non-polar liquids such as benzene and hexane, polymer glasses, or proteins). To explore these phenomena, new types of spectroscopies have been developed by the Fleming Group, including phase control of optical pulses and several photon echo techniques, along with theories describing ultrafast spectroscopies. Current research topics include non-polar solvation, protein-pigment interactions, photosynthetic systems, and multi-dimensional optical spectroscopy.</p>", :scientist_id => 22, :publications_url => "http://www.cchem.berkeley.edu/grfgrp/pages/publications.html")
ScientistProfile.create(:id => 24, :department => "Physical Biosciences Division", :department_url => "http://pbd.lbl.gov/", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Stanley Hall", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-643-3722", :email => "psdehal@lbl.gov", :scientist_id => 19)
ScientistProfile.create(:id => 25, :department => "Physcial Biosciences Division", :department_url => "http://pbd.lbl.gov/", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Stanley Hall", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-643-5683", :email => "amdeutschbauer@lbl.gov", :emphasis => "<p>1) New tools for microbial functional genomics. A fundamental goal in microbiology is the functional annotation of uncharacterized genes identified by genomic sequencing. Due to the anticipated explosion in clonal microbial genome sequences and metagenomes as a result of next-generation sequencing, there is an urgent need for methodologies that rapidly and systematically determine gene function across a range of diverse microorganisms. To meet this challenge, we have created a universal TagModule collection for the parallel analysis of defined mutant collections in any number of microbial species. We are currently assessing the potential of the TagModule collection to assign phenotypes for the majority of genes in the metal-reducing bacterium Shewanella oneidensis MR-1 and in the sulfate-reducing bacterium Desulfovibrio desulfuricans G20.</p>\n<p>2) Microbial systems biology. In collaboration with a number of other research groups, we are engaged in experimental and computational approaches to develop models of microbial metabolism, gene regulation, and signal transduction. As part of the Virtual Institute of Microbial Stress and Survival, we are developing and using next-generation sequencing technology, tiling microarrays, and large mutant libraries to define the transcriptome and determine gene function in the sulfate-reducing bacterium Desulfovibrio vulgaris Hildenborough. As part of the Energy Biosciences Institute, we are using functional genomics in an effort to optimize biofuel production in Zymomonas mobilis ZM4.</p>", :scientist_id => 20, :publications_url => "http://pbd.lbl.gov/10.31.12.backup/resources/Publications/Deutschbauer_Adam_%20CV_2009.pdf")
ScientistProfile.create(:id => 26, :department => "Physical Biosciences Division", :department_url => "http://pbd.lbl.gov/", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "305 Hildebrand Hall", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-643-0225", :phone2_type => "Fax", :phone2 => "510-643-0080", :email => "JDoudna@lbl.gov", :emphasis => "<p>RNA forms a variety of complex globular structures, some of which function like enzymes or form functional complexes with proteins. There are three major areas of focus in the lab: catalytic RNA, the function of RNA in the signal recognition particle and the mechanism of RNA-mediated internal initiation of protein synthesis. We are interested in understanding and comparing catalytic strategies used by RNA to those of protein enzymes, focusing on intron splicing and the self-cleaving RNA from hepatitis delta virus (HDV), a human pathogen. We are also investigating RNA-mediated initiation of protein synthesis, focusing on the internal ribosome entry site (IRES) RNA from Hepatitis C virus. Cryo-EM, x-ray crystallography and biochemical experiments are focused on understanding the structure and mechanism of the IRES and its amazing ability to hijack the mammalian ribosome and associated translation factors. A third area of focus in the lab is the signal recognition particle, which contains a highly conserved RNA required for targeting proteins for export out of cells. Each of these projects seeks to understand the molecular basis for RNA function, using a combination of structural, biophysical and biochemical approaches.</p>", :scientist_id => 21, :publications_url => "http://rna.berkeley.edu/publications.html")
ScientistProfile.create(:id => 70, :department => "Physical Biosciences Division ", :department_url => "http://pbd.lbl.gov", :company => "Lawrence Berkeley National Laboratory ", :address1 => "One Cyclotron Road", :address2 => "Mailstop: 978R4121", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-495-8266", :phone2_type => "Fax", :phone2 => "510-495-2966", :email => "dcchivian@lbl.gov", :emphasis => "<p><span style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">Pursuing an understanding of microbial systems at all scales by computational analysis of the evolution of gene families. How the sequences, structures, and combinations of protein domains vary as a consequence of adaptation to new niches or in the evolution of new function. How combinations of genes act in systems, how genes are transferred between organisms or expanded within a lineage, and ways that sets of genes can be used to identify organisms and characterize function within natural communities, giving rise to ecosystems.</span></p>", :scientist_id => 13, :publications_url => "http://pbd.lbl.gov/resources/publications/dchivian-publications.pdf")
ScientistProfile.create(:id => 86, :department => "Joint BioEnergy Institute", :department_url => "http://jbei.org", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Mailstop: 978R4121", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-495-2694", :phone2_type => "Fax", :phone2 => "510-486-4252", :email => "jlheazlewood@lbl.gov", :emphasis => "<p style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">Dr Heazlewood received his Ph.D in the area of plant molecular biology from <a title=\"La Trobe University\" href=\"http://www.latrobe.edu.au/\">La Trobe University</a> (Australia) in 2000. He then spent several years (2001 - 2003) as a postdoctoral fellow at <a title=\"University of Western Australia\" href=\"http://www.uwa.edu.au/\">The University of Western Australia</a> (Australia) where he used proteomics to investigate plant mitochondria. He was awarded a UWA Postdoctoral Fellowship in 2004 and an <a title=\"Australian Research Council\" href=\"http://www.arc.gov.au/\">Australian Research Council</a> Postdoctoral Fellowship in 2005 to expand this proteomic work into an analysis of protein modifications and signaling pathways in plant mitochondria using phosphoproteomic techniques. During this period (2007) he was also the recipient of an <a title=\"Alexander von Humboldt Foundation\" href=\"http://www.humboldt-foundation.de/web/home.html\">Alexander von Humboldt</a> Research Fellowship to work at the <a title=\"Max Planck Institute of Molecular Plant Physiology\" href=\"http://www.mpimp-golm.mpg.de/\">Max Planck Institute of Molecular Plant Physiology</a> (Germany) using advanced mass spectrometry techniques to further investigate protein modifications in plant mitochondria. Dr Heazlewood was appointed to the Lawrence Berkeley National Laboratory in 2008 at the <a title=\"JBEI\" href=\"http://www.jbei.org/index.shtml\">Joint BioEnergy Institute</a> to investigate the biosynthesis and structure of plant cell walls.</p>\n<p style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">The sequencing of the <a title=\"Arabidopsis Genome Initiative\" href=\"http://www.nature.com/nature/journal/v408/n6814/full/408796a0.html\">Arabidopsis genome</a> in late 2000 revealed that this model plant coded for over 25,000 genes, the vast majority having no known function. The intervening 10 years has seen a considerable investment by the plant community in attempting to determine the functions and interactions of these encoded proteins. Issues such as protein redundancy and the sheer number of unknown function proteins have been major impediments in attributing function using traditional molecular and biochemical techniques. The subcellular location of a protein, functional domain composition and information about co-localizing partners can provide excellent clues to possible biochemical roles of proteins. The rapid development of mass spectrometry based protein identification techniques or 'proteomics' in the early part of this decade has been intensively used to map the subcellular locations of proteins within the plant cell. These studies have provided useful starting points for the allocation of possible function, in defining novel pathways and in demonstrating partitioning of gene families with the eukaryotic cell.</p>\n<div style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\" align=\"center\"><br /><img src=\"http://pbd.lbl.gov/10.31.12.backup/resources/img/about/people/Heazlewood_Subcellular_Image.jpg\" alt=\"\" width=\"600\" height=\"279\" align=\"middle\" hspace=\"12\" /></div>\n<p style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\" align=\"left\"> While defining the mitochondrial proteome of the model plant Arabidopsis thaliana, our interest in multi-gene family partitioning and functional redundancies were stimulated as proteomes from other organelles within the plant were defined. This interest led to the development of the SUBcellular Arabidopsis (<a title=\"SUBA Database\" href=\"http://suba.plantenergy.uwa.edu.au/\">SUBA</a>) Database housing proteomics and fluorescent protein localization data which was used to assess the differential localization of gene families, dual localizations and contamination of organelle preparations.</p>\n<p style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\" align=\"center\"><img src=\"http://pbd.lbl.gov/10.31.12.backup/resources/img/about/people/Heazlewood_Subcellular_Table.jpg\" alt=\"\" width=\"570\" height=\"290\" /></p>\n<p style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\" align=\"left\">Similar approaches are being used at the Lawrence Berkeley National Laboratory in the <a title=\"JBEI\" href=\"http://www.jbei.org/\">Joint BioEnergy Institute</a> to define the Golgi apparatus, the organelle involved in the production of many matrix polysaccharides found in the plant cell wall. Defining known and novel sugar biosynthetic pathways within the Golgi apparatus destined for the cell wall will greatly assist our endeavors to understand the composition and biosynthesis of the plant cell wall enabling future targeted manipulation and exploitation of this biomass for biofuels production.</p>", :scientist_id => 30, :publications_url => "http://www.researcherid.com/rid/A-2554-2008")
ScientistProfile.create(:id => 9, :department => "Physical Biosciences Division", :department_url => "http://pbd.lbl.gov", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Mailstop 64R0121", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-486-4225", :phone2_type => "Fax", :phone2 => "510-486-5909", :email => "PDAdams@lbl.gov", :emphasis => "<table cellspacing=\"0\" cellpadding=\"0\">\r\n\r\n<tr>\r\n<td><span style=\"text-decoration: underline;\">Computational Methods Development</span>&nbsp;<br />Much of my research is focused on the development of&nbsp;<a href=\"http://cci.lbl.gov/\" target=\"_blank\">new algorithms and methods</a>&nbsp;for addressing problems in structural biology. In collaboration with researchers from Los Alamos National Lab, Duke University and Cambridge University we are developing a new software system with NIH Program Project funding, called&nbsp;<a href=\"http://www.phenix-online.org/\" target=\"_blank\">PHENIX</a>, for the automated solution of macromolecular structures using crystallographic methods. With other NIH funding we are also pursuing projects that expand the scope of PHENIX to include the analysis of&nbsp;<a href=\"http://mnc.lanl.gov/\" target=\"_blank\">neutron diffraction data</a>, and&nbsp;<a href=\"http://en.wikipedia.org/wiki/Cryo-electron_microscopy\" target=\"_blank\">single particle cryo-EM reconstruction</a>&nbsp;methods.</td>\r\n<td align=\"right\"><img src=\"http://pbd.lbl.gov/resources/img/about/density_crop.gif\" alt=\"Density Crop\" /></td>\r\n</tr>\r\n<tr>\r\n<td><span style=\"text-decoration: underline;\">Synchrotron Resources for Structural Biology</span><br />As Head of the&nbsp;<a href=\"http://bcsb.lbl.gov/\" target=\"_blank\">Berkeley Center for Structural Biology</a>&nbsp;I oversee the development, maintenance and operation of five synchrotron beamlines for macromolecular crystallographic data collection. We are constantly upgrading the facilities to provide researchers with the best tools for data collection and structure solution. An area of particular interest to me is the structural study of&nbsp;<a href=\"http://www.lbl.gov/Science-Articles/Archive/sb/Nov-2004/05-GroEL-proteins.html\" target=\"_blank\">large macromolecular machines</a>&nbsp;combining multiple experimental and computational methods.</td>\r\n<td align=\"right\"><img src=\"http://pbd.lbl.gov/resources/img/about/822_robot.jpg\" alt=\"Robot\" /></td>\r\n</tr>\r\n<tr>\r\n<td><span style=\"text-decoration: underline;\">Bioenergy Research</span><br />Global warming has become a major world issue and carbon dioxide gas emissions from the burning of fossil fuels are a significant contributor to the problem. The need to develop carbon-neutral and renewable sources of energy has become a priority. The conversion of cellulosic (plant) material to fuels such as ethanol has the potential to provide a significant fraction of fuel in the future. As part of the&nbsp;<a href=\"http://www.jbei.org/\" target=\"_blank\">Joint Bioenergy Institute</a>&nbsp;I am developing new technologies to improve the conversion of biomass to fuels. As part of the&nbsp;<a href=\"http://www.energybiosciencesinstitute.org/\" target=\"_blank\">Energy Biosciences Institute</a>&nbsp;I am collaborating with others to develop new methods for the chemical imaging of plant call wall material.</td>\r\n<td align=\"right\"><img src=\"http://pbd.lbl.gov/resources/img/about/miscanthus.jpg\" alt=\"Miscanthus\" /><br /><br /></td>\r\n</tr>\r\n\r\n</table>", :scientist_id => 1)
ScientistProfile.create(:id => 28, :department => "Physical Biosciences", :department_url => "http://pbd.lbl.gov/", :company => "Lawerence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Lewis Hall ", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-643-0186", :phone2_type => "Fax", :phone2 => "510-486-6059", :email => "JTGroves@lbl.gov", :emphasis => "<p>Physical chemistry of cell membranes -- Principles of molecular organization in cell membranes</p>\n<p>Cells interact with each other and their environment through myriad membrane associated receptors and signaling molecules. In addition to individual receptor-ligand binding, spatial rearrangement of receptors into complex patterns is rapidly emerging as a broadly significant aspect of cell recognition. We are mounting a quantitative investigation of the physical characteristics and principles governing molecular reorganization events during initial stages of cellular recognition and signaling.</p>\n<p>The basic problem to be addressed is the collective interaction between populations of receptors and ligands in two apposed fluid membranes. We wish to understand how the binding kinetics, lateral mobility of the membrane proteins, membrane bending effects, etc. influence molecular organization and recognition. Our approach is aimed toward elucidating how these physico-chemical parameters determine the formation of spatio-temporal patterns at cell signaling junctions. A three-pronged investigative platform which combines novel membrane experiments in reconstituted lipid membranes and cell biology with theoretical calculations and computer simulations has been formulated to meet our goals.</p>\n<p>In the experimental aspect of this project, we are utilizing a variety of supported membrane configurations to examine spontaneous pattern formation in a controlled setting. The use of supported membrane systems is an especially powerful aspect of these studies in that micron and nanometer-scale structures can be fabricated onto the substrate and used to confine and control the fluid membrane in practically any desired geometry. In addition to our studies of processes in cellular membranes, we are developing a variety of hybrid bio-solidstate devices which incorporate fluid membranes. The basic goal is to construct chip-based components which can manipulate, control, and measure membranes and associated molecules. We employ a range of microfabrication techniques including photo- and electron-beam lithography to fabricate micron- and nanometer-scale structures which interface with fluid membranes.</p>", :scientist_id => 26, :publications_url => "http://groveslab.cchem.berkeley.edu/publication.htm")
ScientistProfile.create(:id => 10, :department => "Physical Biosciences", :department_url => "http://pbd.lbl.gov", :company => "University of California: Berkeley", :address1 => "374B Stanley Hall", :address2 => "#3220", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-642-8758", :phone2_type => "Fax", :phone2 => "510-666-2768", :email => "tom@ucxray.berkeley.edu", :summary => "<p>With an eye toward developing new methods, our work focuses on two areas: understanding host-pathogen interactions and discovering the nature and the functions of motions in proteins. Our studies of infectious diseases aim to define mechanisms of signaling and host interactions in TB and HIV/AIDS, two of the major causes of worldwide mortality. Our explorations of protein motions are based on the discovery that X-ray crystallography can reveal a huge ensemble of alternate structures that map out functional dynamics. We use a variety of experimental and computational approaches ranging from synchrotron crystallography and biochemistry to molecular biology and genetics. We also help create multi-scale projects (see Major Initiatives) to bring powerful approaches to bear on critical research problems.</p>", :emphasis => "<p>To cause disease, pathogens sense their environment and manipulate host cell processes. Our work is aimed at defining the mechanisms of sensing and signaling in&nbsp;<em>M. tuberculosis.&nbsp;</em>In addition, we are studying the interactions of HIV proteins with human partners discovered in collaboration with the Zhou (UC Berkeley) and Krogan (UCSF) groups. To explore the fundamental aspects of protein motions, we are developing new methods to model structural ensembles from crystallographic data.</p>\r\n<p><em>Regulatory mechanisms in infectious disease.&nbsp;</em>A major goal is to understand signaling pathways in&nbsp;<em>Mycobacterium tuberculosis (Mtb)</em>. This pathogenic bacterium infects one third of the world's population and causes over two million deaths annually, more than any other infectious disease. Through reversible protein phosphorylation, protein kinases and phosphates provide the fundamental machinery for environmental sensing and physiological signaling. In addition to the traditional two-component signaling systems,&nbsp;<em>Mtb</em>&nbsp;expresses 11 \"eukaryotic-like\" Ser/Thr protein kinases (STPKs) and four protein phosphates that are candidate mediators of developmental changes and host interactions. The biological functions of these systems are largely unknown.</p>\r\n<p>In pioneering structural and mechanistic studies of these Ser/Thr/Tyr signaling systems, we found that universal mechanisms of regulation and substrate recognition govern the functions of prokaryotic and eukaryotic STPKs. These studies also revealed novel mechanisms of regulation, including dimerization of STPK domains and a dynamic lid that protects the PtpB Tyr phosphatase from oxidative inactivation in host cells. Our immediate challenges are to discover the networks of kinase crosstalk in&nbsp;<em>Mtb</em>&nbsp;and to define how the master kinase, PknB, regulates cell-wall homeostasis and cell growth. In these studies, we have used a novel regulated protein knockdown system developed in collaboration with the Rubin (Harvard School of Public Health) and Sassetti (U Mass Medical School) groups to selectively degrade proteins<em>&nbsp;in vivo</em>. Recent work revealed the first structure of a bacterial pseudokinase and showed how phosphorylation of this module regulates the transport of peptidoglycan precursors and the integrity of the cell wall. Our future work focuses on defining the signaling pathways of the&nbsp;<em>Mtb</em>&nbsp;STPKs&nbsp;<em>in vivo</em>&nbsp;and discovering the mechanisms of physiological regulation by post-translational modifications.</p>\r\n<p>The pandemics of TB and AIDS are closely associated, and despite nearly three decades of intensive work, major gaps in knowledge remain about how HIV hijacks host cell functions.&nbsp; Following an unbiased proteomics approach, we worked with the Krogan and Zhou labs to discover new host partners of the viral proteins, including a transcriptional elongation complex recruited to the HIV promoter by HIV Tat. To test the hypothesis that Tat allosterically regulates transcriptional elongation, we are mapping interactions in this large complex and assembling subcomplexes for biochemical and structural studies.</p>\r\n<p><em>Protein polymorphism.&nbsp;</em>Protein function depends on a balance between structural rigidity and flexibility. We developed two new methods, including the Ringer program (<a href=\"http://ucxray.berkeley.edu/ringer.htm\" target=\"_blank\">http://ucxray.berkeley.edu/ringer.htm</a>), for defining conformational polymorphism in protein crystal structures. Our analysis of &gt;400 available structures reveals that native proteins adopt a large number of specific, alternate conformations that previously escaped detection. Our results indicate that crystalline proteins are much more polymorphic than current crystallographic models. In proline isomerase, we found using unorthodox room-temperature X-ray diffraction measurements that coupled shifts define a &ldquo;dynamic network&rdquo; that determines the catalytic rate. Coupled structural changes also may mediate intramolecular signaling pathways, while uncoupled conformations contribute to residual entropy that favors protein folding or ligand binding. Modeling ensembles and exploring their functional significance for enzyme mechanisms, protein recognition and drug binding represent important directions for ongoing work.</p>", :scientist_id => 3)
ScientistProfile.create(:id => 62, :department => "Department of Chemistry", :department_url => "http://chem.berkeley.edu/", :company => "Faculty Scientist", :address1 => "201 Lewis Hall", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-642-0509", :phone2_type => "Fax", :phone2 => "510-486-5283", :email => "MCHammond@lbl.gov", :summary => "<p><span style=\"font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height: 14px;\">Assistant Professor; B.S. California Institute of Technology (2000); HHMI Predoctoral Fellow (2000-05); Ph.D. University of California, Berkeley (2005, Prof. Paul A. Bartlett, advisor); Postdoctoral Fellow, Yale University (2005-09, Prof. Ronald R. Breaker, advisor); Burroughs Wellcome Fund Career Award at the Scientific Interface Investigator (2008-present); National Institutes of Health New Innovator Award (2011-present).</span></p>", :emphasis => "<p><span style=\"font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height: 14px;\">The long-term goals of our research are to investigate the molecular basis of function for natural regulatory RNAs and to learn how to adapt these RNAs for new applications inside cells, including molecular sensing and gene control. We employ organic synthesis, biochemistry, genetics, and bioinformatics to study small molecule-RNA and protein-RNA interactions that are involved in gene regulation in bacteria and plants. Our research aims to provide new insights into the regulation of gene expression in these organisms, and furthermore has applications toward engineering bacteria and plants for biofuel production and other biotechnology projects.</span></p>", :scientist_id => 28, :publications_url => "http://www.cchem.berkeley.edu/mchgrp/publications.html")
ScientistProfile.create(:id => 11, :department => "Bioengineering", :department_url => "http://bioeng.berkeley.edu/index.php", :company => "University of California: Berkeley", :address1 => "Department of Bioengineering", :address2 => "Stanley Hall, Room 308A", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-666-3611", :email => "JCAnderson@lbl.gov", :emphasis => "<p><span style=\"font-family: Arial, Helvetica, sans-serif; font-size: 12px;\">My lab develops the foundational technologies and applications for Synthetic Biology, a ground-up approach to genetic engineering.&nbsp; More information is available at our group&nbsp;</span><a style=\"font-family: Arial, Helvetica, sans-serif; font-size: 12px;\" title=\"Anderson Lab \" href=\"http://andersonlab.qb3.berkeley.edu/\" target=\"_blank\">website</a></p>", :scientist_id => 4)
ScientistProfile.create(:id => 67, :department => "Physical Biosciences", :department_url => "http://pbd.lbl.gov", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Mailstop: Calvin Lab ", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-486-4033", :email => "JCate@lbl.gov", :emphasis => "<p>My lab is exploring the molecular basis for protein biosynthesis, a cellular process that is universally conserved. The ribosome, an RNA and protein factory common to all organisms, is responsible for protein synthesis and plays a key role in the conversion of genotype into phenotype in all forms of life. It translates the genetic code in messenger RNA into the corresponding protein chain. Understanding how the ribosome carries out protein synthesis remains a major challenge in molecular biology. In order to determine the mechanisms underlying protein synthesis and its regulation, my lab uses a combination of x-ray crystallography and biochemical assays to probe ribosome function. We are also developing new technologies that will greatly improve our ability to test the dynamics and energetics of translation in conditions closer to the physiology of the cell.</p>", :scientist_id => 9, :publications_url => "http://lanai.qb3.berkeley.edu/CateLab/Publications.html")
ScientistProfile.create(:id => 68, :department => "Computational and Theoretical Biology Department", :department_url => "http://pbd.lbl.gov/PBD/science/computational_theoretical_biology.html", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Mailstop: Gilman", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-643-6821", :phone2_type => "Fax", :phone2 => "510-643-1566", :email => "DChandler@lbl.gov", :emphasis => "<h4 style=\"font-family: Helvetica, Arial, sans-serif; font-size: 14px; color: #333333; margin: 15px 0px -15px;\">Hydrophobicity at Small and Large Scales</h4>\n<p style=\"font-family: Arial, Helvetica, sans-serif; font-size: 12px;\">In the presence of large hydrophobic solute, the hydrogen bond network in liquid water is necessarily disrupted. This would lead to the depletion of hydrogen bonding and drying near extended surfaces. Professor Chandler and his students, Ka Lum and David Huang, have studied the necessary conditions and the implications of this phenomenon.</p>\n<h4 style=\"font-family: Helvetica, Arial, sans-serif; font-size: 14px; color: #333333; margin: 15px 0px -15px;\">Transition Path Sampling</h4>\n<p style=\"font-family: Arial, Helvetica, sans-serif; font-size: 12px;\">Understanding rare transitions occurring in complex systems, for instance chemical reactions in solution, poses the problem of finding and analyzing the trajectories that move on such a surface from one basin of attraction to another. There are many (possibly torturous) pathways. A representative sampling of all possible routes is required. Together with postdoctorals Peter Bolhuis and Christoph Dellago, and with graduate students Phillip Geissler, Felix Csajka, Gavin Crooks, and Ka Lum, Professor Chandler has developed a systematic approach to find these trajectories with the help of computer simulations.</p>\n<h4 style=\"font-family: Helvetica, Arial, sans-serif; font-size: 14px; color: #333333; margin: 15px 0px -15px;\">Microscopic Theory of Glass Formers</h4>\n<p style=\"font-family: Arial, Helvetica, sans-serif; font-size: 12px;\">On cooling, the viscosity of many super cooled liquids grows by orders of magnitude. Eventually, at low enough temperature, the material becomes so viscous it appears to be a solid, though with no crystalline structure. The material thus formed is called a glass. Except for molecular vibrations over lengths smaller than atomic diameters, the dynamics of the system is arrested. One of the longstanding problems in condensed matter physics is the development of a proper theoretical description of this arrest, and solving this problem is one of Professor Chandler''s current interests. The work he is doing was begun and continues in collaboration with Nottingham physicist Juan P. Garrahan.</p>", :scientist_id => 10, :publications_url => "http://gold.cchem.berkeley.edu/The_Chandler_Group/Publications.html")
ScientistProfile.create(:id => 69, :department => "Physical Biosciences Division", :department_url => "http://pbd.lbl.gov", :company => "Lawrence Berkeley National Laboratory ", :address1 => "One Cyclotron Road", :address2 => "Mailstop: Donner", :city => "Berkeley", :state => "CA", :zip_code => 97420, :phone1 => "510-495-2876", :phone2_type => "Fax", :phone2 => "510-486-7080", :email => "jmchandonia@lbl.gov", :summary => "<table><tbody><tr><td><strong>Projects:</strong></td>\n<td><a title=\"Protein Complex Analysis Project Website\" href=\"http://pcap.lbl.gov/\">Protein Complex Analysis Project (PCAP)</a></td>\n</tr><tr><td>.</td>\n<td><a title=\"ASTRAL Project Website\" href=\"http://astral.berkeley.edu/\">ASTRAL</a></td>\n</tr><tr><td>.</td>\n<td width=\"93%\"><a title=\"SCOP Project Website\" href=\"http://scop.berkeley.edu/\">Structural Classification of Proteins (SCOP)</a></td>\n</tr></tbody></table>", :emphasis => "<p>Structural bioinformatics, in particular:</p>\n<ul><li>Developing the ASTRAL and SCOP databases of protein domain structures and sequences</li>\n<li>Analyzing protein-protein interaction data with the Protein Complex Analysis Project</li>\n<li>Compuational annotation of proteomes, including protein homology prediction</li>\n<li>Structural Genomics target selection strategy</li>\n</ul>", :scientist_id => 11, :publications_url => "http://www.strgen.org/~jmc/publications.php")
ScientistProfile.create(:id => 73, :department => "Department of Chemistry", :department_url => "http://chemistry.berkeley.edu/", :company => "University of California, Berkeley", :address1 => "Department of Chemistry ", :address2 => "125 Lewis Hall", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-642-8545", :email => "MCChang@lbl.gov", :emphasis => "<p><span style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">Our research laboratory utilizes the approaches of mechanistic biochemistry, molecular and cell biology, metabolic engineering, and synthetic biology to address problems in energy and human health. We design and create new biosynthetic pathways in microbial hosts for </span><em style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">in vivo</em><span style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\"> production of biofuels from abundant crop feedstocks and pharmaceuticals from natural products or natural product scaffolds. A unifying theme of all of our projects is a focus on gaining a detailed molecular understanding of how living cells control enzymatic processes within the context of the entire metabolic network. Specific projects under current investigation include (i) the </span><em style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">in vivo</em><span style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\"> production of biofuels from plant biomass, and (ii) the development of new biosynthetic methods for selective, catalytic C-F bond formation under mild conditions.</span></p>", :scientist_id => 12, :publications_url => "http://chem.berkeley.edu/faculty/chang_m/index.php")
ScientistProfile.create(:id => 71, :department => "Biochemical Engineering", :department_url => "http://cheme.berkeley.edu/", :company => "University of California, Berkeley", :address1 => "Department of Biochemical Engineering", :address2 => "Gilman Hall, Room 101A ", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-642-4398", :email => "jwchu@berkeley.edu", :emphasis => "<p><span style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">Therapeutic Discovery, Protein Engineering and Stabilization, Biomedical Devices, Biomaterials, Multiscale Modeling and Simulation Our research program aims to understand how the atomistic scale information of biomolecules (chemistry) governs processes occurring at larger scales in the cell (biology) and applies this knowledge in therapeutic discovery, protein engineering and stabilization, biomaterial science, and the design of molecular devices (engineering). For this purpose, computational methodologies that can be used to propagate information across multiple temporal and spatial domains are being developed and applied to study various biomolecular systems. Current research projects include the allosteric modulation of membrane receptors, mechanical responses of biomolecules, effects of shear on protein conformation and protein-protein interactions, protein stability at oil/water interfaces, and the self-assembly of biomaterials.</span></p>", :scientist_id => 14, :publications_url => "http://cheme.berkeley.edu/faculty/chu")
ScientistProfile.create(:id => 72, :department => "Molecular Foundry", :department_url => "http://foundry.lbl.gov/", :company => "Lawrence Berkeley National Laboratory ", :address1 => "One Cyclotron Road", :address2 => "Mailstop: 67R2206", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-486-4910", :phone2_type => "Fax", :phone2 => "510-486-7268", :email => "schung@lbl.gov", :emphasis => "<p><em style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">A BIO-INSPIRED APPROACH TO TIP-BASED NANOMANUFACTURING</em><span style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">: </span><br style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\" /><br style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\" /><span style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">Developing generic platforms to create deterministic patterns of nanostructures at surface is one of the central challenges of nanotechnology. To address this challenge, tip-based nanofabrication (TBN) has been developed to fabricate nanowires and nanoparticles of which TBN relies on patterning of functional biotemplating molecules (such as RNA aptamers, peptides etc) selected for their ability to mediate material-specific formation of inorganic solids (i.e. metallic and semiconducting) under mild reaction conditions.</span></p>", :scientist_id => 15, :publications_url => "http://pbd.lbl.gov/10.31.12.backup/resources/Publications/Chung_Sungwook%20CV_LBNL.pdf")
ScientistProfile.create(:id => 75, :department => "Advanced Biological and Environmental X-Ray Spectroscopy (ABEX)", :department_url => "http://abex.lbl.gov/", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Mailstop: 6-2100", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-486-4720", :phone2_type => "Fax", :phone2 => "510-486-5664", :email => "SPCramer@lbl.gov", :emphasis => "<h4 style=\"font-family:Helvetica, Arial, sans-serif;font-size:14px;color:#333333;margin:15px 0px -15px;text-decoration:underline;\">Spectroscopy Methods Development and Applications</h4>\n<p> </p>\n<p style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">The Cramer group uses spectroscopic methods to characterize the molecular and electronic structure of metal sites in bioinorganic systems and environmental samples. Of particular interest are the enzymes nitrogenase (which catalyzes nitrogen fixation to ammonia) and hydrogenases (which catalyze interconversion of protons and electrons to hydrogen molecules). The spectroscopic methods employed include soft x-ray spectroscopy and XMCD at the ALS, as well as EXAFS, NRVS, and other synchrotron techniques at higher energy sources. On the ALS mezzanine and at UC Davis, we also employ conventional methods such as FT-IR, Raman, EPR/ENDOR, and Mössbauer spectroscopies.</p>\n<h4 style=\"font-family:Helvetica, Arial, sans-serif;font-size:14px;color:#333333;margin:15px 0px -15px;text-decoration:underline;\">ABEX</h4>\n<p> </p>\n<p style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">We are funded by DOE Office of Biological and Environmental Research to operate a user resource that enables chemists and biologists to more easily use soft x-ray spectroscopy. Examples of our work include development of x-ray magnetic circular dichrolism (XMCD) end stations and superconducting tunnel junction (STJ) detectors.</p>", :scientist_id => 17, :publications_url => "http://pbd.lbl.gov/10.31.12.backup/resources/Publications/steve_cramer_publications.pdf")
ScientistProfile.create(:id => 76, :department => "Physical Biosciences", :department_url => "http://pbd.lbl.gov", :company => "Lawrence Berkeley National Laboratory", :address1 => "One Cyclotron Road", :address2 => "Mailstop: Stanley", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-219-5039", :email => "gecrooks@lbl.gov", :emphasis => "<p><span style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">My research efforts, in recent years, has sought to understand the application of new insights in the thermodynamics of small systems away from equilibrium to interesting problems in biology and physics. The goal is to develop our fundamental understanding of nanoscale molecular machines, both biological and artificial. We are particularly interested in discovering design principles that will allow us to engineer and customize our own efficient and effective molecular gadgetry. We wish to understand the thermodynamic efficiency of molecular machines, the tradeoff of efficiency with power output, how to determine the optimal thermodynamic processes, and how to engineer machines that can approach these optimal performances</span></p>", :scientist_id => 18, :publications_url => "http://threeplusone.com/gec/pubs")
ScientistProfile.create(:id => 77, :department => "Computational and Theoretical Biology", :department_url => "http://pbd.lbl.gov/PBD/science/computational_theoretical_biology.html", :company => "Faculty Scientist", :address1 => "One Cyclotron Road", :address2 => "Donner Hall, Room 0472", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-486-7365", :phone2_type => "Fax", :phone2 => "510-486-5342", :email => "TLHead-Gordon@lbl.gov", :emphasis => "<p><span style=\"font-family:Arial, Helvetica, sans-serif;font-size:12px;\">The THG group combines experimental, theoretical and computational approaches toward the problems of protein folding, structure prediction, aqueous hydration, and structural biology. Our group's research is highly multidisciplinary and collaborative among the areas of structural biology, physical chemistry, physics, computer science and mathematical optimization.</span></p>", :scientist_id => 29, :publications_url => "http://thglab.berkeley.edu/")
ScientistProfile.create(:id => 20, :department => "Physical Biosciences Division", :department_url => "http://pbd.lbl.gov/", :company => "Lawrence Berkeley National Laboratory", :address1 => "Energy Biosciences Building (EBB)", :address2 => "2151 Berkeley Way, 5th Floor", :city => "Berkeley", :state => "CA", :zip_code => 94704, :phone1 => "510-495-2366", :phone2_type => "Fax", :phone2 => "510-486-6219", :email => "APArkin@lbl.gov", :emphasis => "<p>The Arkin Lab works on detailed modeling of genetic and biochemical networks with emphasis on developmental systems. The laboratory creates custom genetic circuitry in Saccaromyces cerivisiae and multichannel, protein and small molecule biosensors. The Arkin Lab is interested in the detailed physical analysis of the network of biochemical and genetic reactions that govern cellular development. The goal is to divine the engineering principles of the control systems that determine cell behavior and differentiation in response to internal and external signals. Because of their simplicity (relative to eukaryotic cells), and because many bacterial genome sequencing projects have recently completed, we study mostly bacterial and viral circuitry. Particular biological systems currently under study in my lab include, l-phage/Escherichia coli interactions, the role of stochastic phase-variation of type-1 pili in uropathic E. coli virulence, and analysis of the sporulation initiation and germination pathways in Bacillus subtilis. As the basis for such analyses we examine the detailed mechanisms of the underlying chemical reactions. For example, a rigorous physical analysis of the mechanisms of prokaryotic gene expression revealed that the temporal pattern of protein production from a single gene is an erratic and bursty stochastic process. Analysis of networks of such genes responsible for developmental switches demonstrated that while some architectures generate deterministic outcomes despite this noise, others exploit the noise to produce population diversity to, for example, evade attack by the immune system. In addition to theoretical analyses, the laboratory has started experimental measurements on such systems and has begun design and implementation (in yeast and E. coli) of our own custom genetic circuitry. Thus, the laboratory applies theoretical and computational analyses from dynamical systems, stochastic processes, chemical kinetics and statistical mechanics and methods from molecular biology to determine the principles of cellular signal processing and to aid in design of custom cellular circuitry that may, for example, act as sensitive biosensors.</p>", :scientist_id => 5, :publications_url => "http://bioeng.berkeley.edu/gradfaculty/gradcv/aarkin.php")
ScientistProfile.create(:id => 61, :prefix => "Dr.", :department => "Physical Biosciences Division ", :department_url => "http://pbd.lbl.gov", :company => "Lawrence Berkeley National Laboratory ", :address1 => "One Cyclotron Road", :address2 => "Mailstop: 6R2100 ", :city => "Berkeley", :state => "CA", :zip_code => 94720, :phone1 => "510-486-5378", :phone2_type => "Fax", :phone2 => "510-486-5298", :email => "mhammel@lbl.gov", :emphasis => "<ol style=\"font-family: Arial, Helvetica, sans-serif; font-size: 12px;\">\n<li style=\"margin-bottom: 10px;\"><em>Dynamic structure of DNA repair and Replication machinery<br /></em><br />Combining Crystallography and Small angel X-ray scattering (SAXS) to characterize functionally-important dynamic conformational and assembly trajectories in solution. High fidelity DNA synthesis is beneficial for maintaining genetic information and transactions involving DNA synthesis required to faithfully replicate. Diversity in the replication machineries and the sequence complexity of genomes provide the potential to DNA synthesis error. There for the DNA replication is tightly bound to the DNA repair. Our current understanding of initiation of DNA replication could be understood as a hierarchical combination of different factors, where these factors are sharing significant role in the replication and repair tasks. The gross conformational changes/flexibility associated with its activation, DNA binding and structural arrangement of those factors has been so far the missing information in the structural biology of biogenesis of the DNA replication-repair. We have successfully applied the combination of crystallography and SAXS technique for various d DNA replication- repair proteins.<br /><br />Within the Structural Biology of DNA Repair (SBDR) &ndash; projects we determined several dynamic structures of protein complexes involved in DNA replication-repair: Ku/DNA-Pkcs (Hammel et al., 2009), mPNK/DNA (Bernstein et al., 2009), DNA-Polymerase Delta (Pols) &delta; (Jain et al., 2009).</li>\n<li style=\"margin-bottom: 10px;\"><a title=\"Published Paper \" href=\"http://bl1231.als.lbl.gov/2009/07/robust_high-throughput_solutio.php\" target=\"_blank\"><em>Robust, High-Throughput Solution Structural Analysis by SAXS</em></a><strong><br /></strong><br />We&rsquo;re developing an efficient pipeline enabling high-throughput analysis of protein structure in solution with small angle X-ray scattering (SAXS). Our SAXS pipeline combines automated sample handling of microliter volumes, temperature and anaerobic control, rapid data collection and data analysis, and couples structural analysis with automated archiving. SAXS analysis allowed us to distinguish aggregated and unfolded proteins, define global structural parameters and oligomeric states for most samples, identify shapes and similar structures for 25 unknown structures, and determine envelopes for 41 proteins. We believe that high-throughput SAXS is an enabling technology that may change the way that structural genomics research is done.&nbsp;<br /><br />Hura GL, Menon AL, Hammel M, Rambo RP, Poole II FL, Tsutakawa SE, Jenney Jr FE, Classen S, Frankel KA, Hopkins RC, Yang S, Scott JW, Dillard BD, Adams MWW, &amp; Tainer JA, \"<a title=\"Article Abstract\" href=\"http://www.nature.com/nmeth/journal/v6/n8/abs/nmeth.1353.html\" target=\"_blank\">Robust, high-throughput solution structural analyses by small angle X-ray scattering (SAXS)</a>\" Nature Methods. 2009 Aug; 6(8):606-12.</li>\n</ol>", :scientist_id => 27, :publications_url => "http://pbd.lbl.gov/10.31.12.backup/resources/Publications/Hammel_Michal_CV.pdf")
